llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -2755,7 +2758,7 @@ void VPlanTransforms::narrowInterleaveGroups(VPlan &Plan, ElementCount VF,
       auto *L = new VPWidenLoadRecipe(
           *cast<LoadInst>(LoadGroup->getInterleaveGroup()->getInsertPos()),
           LoadGroup->getAddr(), LoadGroup->getMask(), /*Consecutive=*/true,
-          /*Reverse=*/false, LoadGroup->getDebugLoc());
+          /*Reverse=*/false, {}, LoadGroup->getDebugLoc());
```

Comment:
The single widenLoad/StoreRecipe in this case drops the metadata of its insertPos/underlying load/store? Seems an exception worthy of a note, perhaps an explicit subsequent dropMetadata() call?

Reply:
This change is no longer needed?